src/test/java/uk/gov/ch/oauth/Oauth2Test.java

```diff
@@ -45,14 +52,18 @@
 @ExtendWith(MockitoExtension.class)
 public class Oauth2Test {
 
-    public static MockWebServer mockServer;
+    private static final String CODE = "code";
+    private static final String NONCE = "noncevalue";
+    private static final String STATE = "state";
     private static OAuthToken oauthToken;
+    private static UserProfileResponse userProfile;
     private final SignInInfo signInInfo = new SignInInfo();
+    public static MockWebServer mockServer;
```

Comment:
Why are these non constants static?

Reply:
Why is SignIn initiated but UserProfile and Auth token aren't?

Reply:
I believe `MockWebServer` has to be static, which is why I made `userProfile` & `oauthToken` as they are being initialised within the setup() which is a static method.